platform/android/MapboxGLAndroidSDK/build.gradle

```diff
@@ -16,9 +16,10 @@ dependencies {
     }
 
     // Mapbox Android Services (Telemetry support)
-    compile(rootProject.ext.dep.mapboxAndroidTelemetry) {
-        transitive = true
-    }
+    compile(name: 'libandroid-telemetry', ext: 'aar')
```

Comment:
If you're changing the telemetry library too, we'll need the equivalent PR on `mapbox-java`.